internal/component/common/config/types.go

```diff
@@ -92,6 +93,7 @@ func (h *HTTPClientConfig) Convert() *config.HTTPClientConfig {
 		FollowRedirects: h.FollowRedirects,
 		EnableHTTP2:     h.EnableHTTP2,
 		ProxyConfig:     h.ProxyConfig.Convert(),
+		Host:            h.Host,
```

Comment:
@jkroepke It looks like `Host` [got removed](https://github.com/prometheus/common/pull/597) from the upstream repo.

Reply:
yes,  see https://github.com/prometheus/common/pull/645